src/ui/Header/Sidebar/AllTabs/Show-More.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import SideTag from "../../../SideTag";
+import "./index.css";
+import { ReactComponent as ShowMoreIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/icons/baricons/show-more.svg";
```

Comment:
this import is incorrect, this is a local file and not within the project folder